lib/telephony.rb

```diff
@@ -71,6 +72,19 @@ def self.send_confirmation_otp(to:, otp:, expiration:, otp_format:,
     ).send_confirmation_otp
   end
 
+  def self.send_notification(message:, to:, expiration:,
+                             channel:, domain:, country_code:, extra_metadata:)
+    NotificationSender.new(
+      message: message,
+      to: to,
+      expiration: expiration,
+      channel: channel,
```

Comment:
for my understanding - what is channel?

Reply:
looks like delivery method, either sms or voice.